### PR TITLE
add doap file for blabber.im

### PR DESCRIPTION
**Pull request by kriztan**

### CODE REVIEW

**Review by Echolon — APPROVED** [review]
Many thanks for taking the path till here!

File: data/clients.json [hunk 1]
```diff
@@ -61,8 +61,8 @@
         "url": "https://www.bitlbee.org"
     },
     {
-        "doap": null,
-        "last_renewed": "2020-11-28T00:00:00",
+        "doap": "https://codeberg.org/kriztan/blabber.im/src/branch/master/blabber.im.doap",
```
Comment (wurstsalat3000) [inline data/clients.json:64]:
```suggestion
        "doap": "https://codeberg.org/kriztan/blabber.im/raw/branch/master/blabber.im.doap",
```

**Review by wurstsalat3000 — COMMENTED** [review]
Please also change the following, since the DOAP file provides these infos:

```
"platforms": [],
"url": null
```

**Comment by wurstsalat3000** [comment]
.. too late. I'll change that myself.

**Pull request merged**